api/v1/qdrantcluster_types.go

```diff
@@ -845,6 +882,51 @@ type NodeStatus struct {
 	// Version specifies the version of Qdrant running on the node
 	// +optional
 	Version string `json:"version,omitempty"`
+
+	// Status phase of the Pod of the node
+	// +optional
+	PodPhase corev1.PodPhase `json:"podPhase,omitempty"`
+	// Conditions of the Pod of the node
+	// +optional
+	PodConditions []corev1.PodCondition `json:"podConditions,omitempty"`
+	// Status message of the Pod of the node
+	// +optional
+	PodMessage string `json:"podMessage,omitempty"`
+	// Status reason of the Pod of the node
+	// +optional
+	PodReason string `json:"podReason,omitempty"`
+	// Details container statuses of the Pod of the node
+	// +optional
+	ContainerStatuses []corev1.ContainerStatus `json:"containerStatuses,omitempty"`
+	// Recent Kubernetes Events related to the Pod of the node
```

Comment:
What is recent? When will these be removed again, imo this shouldn't be an ever growing array.